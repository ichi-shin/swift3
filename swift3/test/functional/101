#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-8_6

. ./common

S3USER=tester

_s3_put /bucket
_s3_put /bucket/obj -T /dev/null -H "x-amz-storage-class: REDUCED_REDUNDANCY" -D - | _filter_curl xml
_s3_get /bucket/obj -D - | _filter_curl xml

