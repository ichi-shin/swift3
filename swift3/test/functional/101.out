QA output created by 101
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -T /dev/null -H x-amz-storage-class: REDUCED_REDUNDANCY -D - http://SWIFT_HOST/bucket/obj... 400
HTTP/1.1 400 Bad Request
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>InvalidStorageClass</Code>
  <Message>The storage class you specified is not valid.</Message>
  <RequestId>TXID</RequestId>
</Error>
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/bucket/obj... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <RequestId>TXID</RequestId>
  <Key>obj</Key>
</Error>
